plugins/permission-backend/src/service/PermissionIntegrationClient.test.ts

```diff
@@ -91,7 +94,7 @@ describe('PermissionIntegrationClient', () => {
     });
 
     it('should make a POST request to the correct endpoint', async () => {
-      await client.applyConditions('plugin-1', [
+      await client.applyConditions('plugin-1', mockCredentials.none(), [
```

Comment:
For a test like this it might be helpful to switch the default credentials in `mockServices.auth({ ... })` to `none()`